src/sysinfo.cc

```diff
@@ -74,8 +74,10 @@ bool ReadIntFromFile(const char* file, long* value) {
   if (fd != -1) {
     char line[1024];
     char* err;
+    ssize_t read_err;
     memset(line, '\0', sizeof(line));
-    CHECK(read(fd, line, sizeof(line) - 1));
+    read_err = read(fd, line, sizeof(line) - 1);
```

Comment:
Please declare `read_err` on the same line you initialize it, not before.

Also `read_err` is only used in debug mode, so in release mode we'll get a unused variable error about it. To work around this please add `((void)read_err); // prevent unused warning`